eth/provider/provider.go

```diff
@@ -21,10 +21,13 @@
 package provider
 
 import (
+	"github.com/ethereum/go-ethereum/cmd/utils"
+	"github.com/ethereum/go-ethereum/eth/ethconfig"
 	"github.com/ethereum/go-ethereum/node"
 
 	"pkg.berachain.dev/polaris/eth/api"
 	"pkg.berachain.dev/polaris/eth/core"
+	"pkg.berachain.dev/polaris/eth/graphql"
```

Comment:
why do we need to include the graphql folder in polaris? can we just import from geth?

Reply:
ya lemme fix

Reply:
we can reuse it all from geth